core/src/commonMain/kotlin/Kord.kt

```diff
@@ -365,6 +370,63 @@ public class Kord(
         scheduledEventId: Snowflake? = null,
     ): Invite? = with(EntitySupplyStrategy.rest).getInviteOrNull(code, withCounts, withExpiration, scheduledEventId)
 
+    /**
+     * Requests to get all [Sku]s for this application.
+     *
+     * @throws [RequestException] if anything went wrong during the request.
```

Comment:
```suggestion
     * @throws [RestRequestException] if anything went wrong during the request.
```

Reply:
Should I edit the kdoc of the other methods I created?

Reply:
only those that only use rest, otherwise they could throw another subclass of `RequestException`

Reply:
good to know 👍🏼